### ISSUE TITLE
Chapter 3 "Serializing models themselves" is missing

### ISSUE THREAD

**Issue by lupesko** [zackchase/mxnet-the-straight-dope#264]
Chapter 3 has a section "Serializing models themselves", which is missing content.
See [here](http://gluon.mxnet.io/chapter03_deep-neural-networks/serialization.html#Serializing-models-themselves).

It needs to address the flow for:
- How do you serialize the model (network arch, not weights)
- How do you load a serialized model?
- Are there any restrictions? (non-hybridizable blocks)

@srochel @zackchase @mli

**Comment by zackchase** [comment]
Yeah I need to move the serializing models part into hybridize. @piiswrong @mli - do we have a way yet to serialize traces through imperative code without explicitly running symbols through hybrid blocks?
